modules/dynamic-provisioning-available-plugins.adoc

```diff
@@ -27,11 +27,11 @@ configured provider's API to create new storage resources:
 tag each node with `Key=kubernetes.io/cluster/<cluster_name>,Value=<cluster_id>` 
 where `<cluster_name>` and `<cluster_id>` are unique per cluster.
 
-//|GCE Persistent Disk (gcePD)
-//|`kubernetes.io/gce-pd`
-//|In multi-zone configurations, it is advisable to run one {product-title} 
-//cluster per GCE project to avoid PVs from getting created in zones where 
-//no node from current cluster exists.
+|GCE Persistent Disk (gcePD)
+|`kubernetes.io/gce-pd`
+|In multi-zone configurations, it is advisable to run one {product-title}
+cluster per GCE project to avoid PVs from getting created in zones where
+no node from current cluster exists.
```

Comment:
from current cluster -> from the current cluster?

Reply:
Adjusted this sentence to make it read a bit cleaner.